python/ray/_private/worker.py

```diff
@@ -461,6 +461,8 @@ def __init__(self):
         # Create the lock here because the serializer will use it before
         # initializing Ray.
         self.lock = threading.RLock()
+        # By default, don't show logs from other drivers.
+        self._filter_logs_by_job = True
```

Comment:
this is super hacky, so please update the comment to indicate why it's here and point at this PR :)